src/components/Form/Form.js

```diff
@@ -0,0 +1,139 @@
+import { useState } from 'react';
+import StyledForm from '../styled-components/StyledForm';
+import StyledInput from '../styled-components/StyledInput';
+import StyledLabel from '../styled-components/StyledLabel';
+import useStore from '../../hooks/useStore';
+import StyledButton from '../styled-components/StyledButton';
+
+export default function Form() {
+	const [titleValue, setTitleValue] = useState('');
+	const [adressValue, setAdressValue] = useState('');
+	const [datumValue, setDatumValue] = useState('');
+	const [zeitraumValue, setZeitraumValue] = useState('');
+	const [anzahlflaschenValue, setAnzahlflaschenValue] = useState('');
```

Comment:
Try to adapt your state names to your now english variable names, then this part will be more understandable.